lms/templates/page_banner.html

```diff
@@ -0,0 +1,24 @@
+## mako
+
+<%page expression_filter="h"/>
+
+<%namespace name='static' file='static_content.html'/>
+<% online_help_token = self.online_help_token() if hasattr(self, 'online_help_token') else None %>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+from openedx.core.djangoapps.util.user_messages import user_messages
+%>
+
+% if user_messages:
+    <div class="page-banner">
+        <div class="user-messages">
+            % for message in user_messages(request):
+                <div class="alert ${HTML(message.css_class)}" role="alert">
```

Comment:
The code needs to escape on the other side, because by the time it reaches this code it is no longer an HTML object. I realized looking at this that I don't need to be escaping the css_class though. It is only the `message_html` field that must be assumed to be properly escaped HTML.

I didn't have this logic working correctly, but I do now after writing the test for it.